build.gradle

```diff
@@ -48,6 +56,11 @@ patchPluginXml {
     }
 }
 
+tasks.markdownToHtml.dependsOn("createReleaseNotes")
```

Comment:
I am chaining tasks so that each task can operate under expected conditions. (It's hard to change html, when markdown hasn't been converted to html yet!)